v1 (#36)
cosmos/new-cosmos.network: content/articles/what-is-blockchain.md

alt: Article 10
author: 
  name: SAMUEL L. IPSUM
  bio: Your bones don't break, mine do. That's clear. Your cells react to bacteria and viruses differently than mine. You don't get sick, I do. That's also clear. But for some reason, you and I react the exact same way to water. We swallow it too fast, we choke. We get some in our lungs, we drown. However unreal it may seem, we are connected, you and I. We're on the same curve, just on opposite ends.
  img: https://images.unsplash.com/photo-1516331138075-f3adc1e149cd?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=800&q=60
tags: 
  - blockchain
---

A blockchain is a ledger securely replicated across many machines. By distributing the same ledger across hundreds or thousands of machines, blockchains are more resilient to downtime events like power outages, hardware faults, natural disasters, and geopolitical censorship. To form consensus, these machines regularly collaborate to determine which sets of new transactions (“blocks”) are valid. These blocks of transactions link together to form a blockchain.

> This technology enables the development of permissionless decentralized applications. In other words, blockchains allow the creation of a new kind of digital application that do not require a central coordinator to operate. Think electronic money without central banks, digital asset trading without centrally-operated stock exchanges, or social network platforms without central administrators.

Reviewer: ```suggestion
This technology enables the development of permissionless decentralized applications. In other words, blockchains allow the creation of a new kind of digital app that does not require a central coordinator to operate. Think electronic money without central banks, digital asset trading without centrally-operated stock exchanges, or social networking platforms without central administrators.
```
